packages/manager/modules/pci/src/projects/project/storages/databases/database/replications/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,19 @@
+{
+  "pci_databases_replications_tab_title": "Flux de réplication",
+  "pci_databases_replications_tab_description": "Les flux de réplication permettent la copie automatique des topics de données et de la configuration entre deux clusters Kafka. Un service MirrorMaker peut exécuter plusieurs flux de réplication.",
```

Comment:
```suggestion
  "pci_databases_replications_tab_description": "Les flux de réplication permettent la copie automatique des topics de données et de la configuration entre deux clusters Kafka. Un service Mirror Maker peut exécuter plusieurs flux de réplication.",
```

Reply:
Probably, but as said before, I have to use the exact text given by content team.